src/Cli/Extras/Process.hs

```diff
@@ -61,6 +62,10 @@ import Control.Monad.Log (Severity (..))
 import Cli.Extras.Logging (putLog, putLogRaw)
 import Cli.Extras.Types (CliLog, CliThrow)
 
+#if !(MIN_VERSION_base(4, 13, 0))
+import Control.Monad.Fail (MonadFail)
+#endif
```

Comment:
@madeline-os should I use these CPP guards to only import MonadFail when needed, should I disable the "redundant imports" warning, or should I just leave the warning there?

Reply:
Go for CPP guards, or just leave the warning in there if it already exists.